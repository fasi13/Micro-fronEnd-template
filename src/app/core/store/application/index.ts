export * from './application.effects';
export * from './application.actions';