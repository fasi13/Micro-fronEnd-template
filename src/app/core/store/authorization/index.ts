export * from './authorization.effects';
export * from './authorization.actions';