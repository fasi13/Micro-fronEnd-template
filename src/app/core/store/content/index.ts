export * from './content.effects';
export * from './content.actions';
