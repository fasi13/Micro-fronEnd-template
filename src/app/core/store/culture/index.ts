export * from './culture.actions';
export * from './culture.effects';
