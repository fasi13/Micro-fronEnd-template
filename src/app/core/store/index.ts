export * from './authorization';
export * from './router';
export * from './application';
export * from './store.reducers';