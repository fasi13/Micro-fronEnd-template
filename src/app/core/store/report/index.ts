export * from './report.actions';
export * from './report.effects';
