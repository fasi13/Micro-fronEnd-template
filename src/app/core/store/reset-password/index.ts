export * from './reset-password.effects';
export * from './reset-password.actions';
