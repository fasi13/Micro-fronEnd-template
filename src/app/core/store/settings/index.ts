export * from './settings.actions';
export * from './settings.effects';
