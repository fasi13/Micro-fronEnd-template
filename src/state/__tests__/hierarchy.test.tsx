import { useSearchStore } from '../searchState.store';

const { setSearchLoading } = useSearchStore();
describe('adds 1 + 2 to equal 3', () => {
	it('should contain a function', () => {
		expect(setSearchLoading).toBe('function');
		expect(true).toBe(true);
	});
});
