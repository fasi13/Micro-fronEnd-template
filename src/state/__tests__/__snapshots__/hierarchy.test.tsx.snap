// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`hieararchy store hierarchy store is defined 1`] = `
Object {
  "activeNodeId": 0,
  "createApplication": [Function],
  "createApplicationGroup": [Function],
  "defaultExpandLevel": 0,
  "editApplication": [Function],
  "editApplicationGroup": [Function],
  "error": "",
  "getUserApplication": [Function],
  "hierarchyData": Array [],
  "initializeHierarchyState": [Function],
  "loadApplication": [Function],
  "loading": false,
  "setError": [Function],
  "setLoading": [Function],
  "setLoadingChildren": [Function],
  "setNodeError": [Function],
  "setSaving": [Function],
  "toggleCollapse": [Function],
  "toggleEdit": [Function],
  "toggleNewEditor": [Function],
}
`;
